lib/absinthe/phase/document/arguments/data.ex

```diff
@@ -53,7 +53,12 @@ defmodule Absinthe.Phase.Document.Arguments.Data do
   def handle_node(%Input.Value{normalized: %Input.Object{fields: fields}} = node) do
     data =
       for field <- fields, include_field?(field), into: %{} do
-        {field.schema_node.identifier, field.input_value.data}
+        # Scalar child nodes will not have a schema_node
+        if field.schema_node != nil do
+          {field.schema_node.identifier, field.input_value.data}
+        else
+          {String.to_existing_atom(field.name), field.input_value.data}
```

Comment:
For security reasons, I would assume that we can only support atom keys that have been defined in code at some point

Reply:
I'm not entirely convinced we don't want to just keep it as a string to be honest. Atoms should be reserved for input known to be good, but Absinthe has no way of doing so with these type of scalars. They're much more like traditional HTTP params. I also think limiting folks to field names that appear elsewhere in the code base is going to be an issue.

Reply:
alright I left this as a string which I think is the right call